cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -29,14 +30,70 @@
         gen_func=providers,
         filters=[ProviderFilter(
             classes=[InfraProvider, CloudProvider],
-            required_fields=['provisioning'])])
+            required_fields=['provisioning'])]),
+    pytest.mark.usefixtures("setup_provider")
 ]
 
 
+@pytest.fixture(scope='module')
+def new_role(appliance):
+    role = appliance.collections.roles.instantiate(name='EvmRole-user_self_service')
+    copied_role = role.copy(name="{role}_{name}".format(
+        role=role.name, name=fauxfactory.gen_alpha()), restriction="None")
+
+    yield copied_role
+    copied_role.delete_if_exists()
+
+
+@pytest.fixture(scope='module')
+def new_group(appliance, new_role):
+    collection = appliance.collections.groups
+    group = collection.create(description='group_{}'.format(fauxfactory.gen_alphanumeric()),
+                              role=new_role.name,
+                              tenant=appliance.collections.tenants.get_root_tenant().name)
+    yield group
+    group.delete_if_exists()
+
+
+@pytest.fixture(scope='module')
+def new_user(appliance, new_group, new_credential):
```

Comment:
I see fixtures of new_user , new_group and new_role used in lot of tests.
Let's create fixture for these in common place .